rotkehlchen/db/upgrades/v39_v40.py

```diff
@@ -64,6 +73,31 @@ def _purge_kraken_events(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _reset_kraken_events')
 
 
+def _add_base_chain_data(write_cursor: 'DBCursor') -> None:
+    log.debug('Enter _add_base_chain_data')
+    write_cursor.execute('INSERT OR IGNORE INTO location(location, seq) VALUES ("j", 42);')
+    write_cursor.executemany(
+        'INSERT OR IGNORE INTO rpc_nodes(name, endpoint, owned, active, weight, blockchain) '
+        'VALUES (?, ?, ?, ?, ?, ?)',
+        DEFAULT_BASE_NODES_AT_V40,
+    )
+    log.debug('Exit _add_base_chain_data')
+
+
+def _update_optimism_transactions_table(write_cursor: 'DBCursor') -> None:
```

Comment:
Ah damn sorry I am an idiot. I just realized after looking at the code again this upgrade step is not needed. `tx_id` references the `identifier` of `evm_transactions`. Which combines tx_hash + chain_id. So chain id is not needed here.